lib/call-session.js

```diff
@@ -497,16 +497,19 @@ class CallSession extends Emitter {
           };
         }
         else {
+          const uri = parseUri(this.req.uri);
           hdrs = {
             ...hdrs,
             'From': createBLegFromHeader({
               logger: this.logger,
               req: this.req,
+              transport: uri.params.transport,
```

Comment:
have you tested the case where there is no transport param?  Should we just use the `...(uri.params.transport && ..` pattern?